include/grid_map_geo/grid_map_geo.hpp

```diff
@@ -171,11 +170,12 @@ class GridMapGeo {
    */
   void AddLayerNormals(std::string reference_layer);
 
+  geometry_msgs::msg::TransformStamped static_transformStamped;
```

Comment:
```suggestion
  geometry_msgs::msg::TransformStamped static_transformStamped_;
```

Nit: Google style guide for class member var

Reply:
Done!